### PR TITLE
[NPU]fix eplb with nz

### PR DESCRIPTION
**Pull request by amote-i**
## Motivation

fix eplb error when weight format type is ACL_FORMAT_FRACTAL_NZ

## Modifications

replace the big nz tensor with List<nz_tensor>

## Accuracy Tests

error before fix

image

accuracy after fix

image

## Checklist

- [x] Format your code according to the [Format code with pre-commit](https://docs.sglang.io/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [x] Add unit tests according to the [Run and add unit tests](https://docs.sglang.io/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [x] Update documentation according to [Write documentations](https://docs.sglang.io/developer_guide/contribution_guide.html#write-documentations).
- [x] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.io/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.io/developer_guide/contribution_guide.html#benchmark-the-speed).
- [x] Follow the SGLang code style [guidance](https://docs.sglang.io/developer_guide/contribution_guide.html#code-style-guidance).

## Review Process

1. Ping Merge Oncalls to start the PR flow. See the [PR Merge Process](https://github.com/sgl-project/sglang/blob/main/.github/MAINTAINER.md#pull-request-merge-process).
2. Get approvals from [CODEOWNERS](https://github.com/sgl-project/sglang/blob/main/.github/CODEOWNERS) and other reviewers.
3. Trigger CI tests with [comments](https://docs.sglang.io/developer_guide/contribution_guide.html#how-to-trigger-ci-tests) or contact authorized users to do so.
   - `/tag-run-ci-label`, `/rerun-failed-ci`, `/tag-and-rerun-ci`
4. After green CI and required approvals, ask Merge Oncalls to merge.

### CODE REVIEW

File: python/sglang/srt/hardware_backend/npu/quantization/fused_moe_method_npu.py [hunk 1]
```diff
@@ -183,8 +183,10 @@ def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
                 requires_grad=False,
             )
 
-        layer.w13_weight.data = npu_format_cast(layer.w13_weight.data)
-        layer.w2_weight.data = npu_format_cast(layer.w2_weight.data)
+        for i in range(layer.w13_weight.data.shape[0]):
```
Comment (iforgetmyname) [inline python/sglang/srt/hardware_backend/npu/quantization/fused_moe_method_npu.py]: add comments

File: python/sglang/srt/hardware_backend/npu/quantization/fused_moe_method_npu.py [hunk 2]
```diff
@@ -326,8 +328,10 @@ def process_weights_after_loading(
             layer.w2_weight.data.transpose(1, 2).contiguous(), requires_grad=False
         )
 
-        layer.w13_weight.data = npu_format_cast(layer.w13_weight.data)
-        layer.w2_weight.data = npu_format_cast(layer.w2_weight.data)
+        for i in range(layer.w13_weight.data.shape[0]):
```
Comment (iforgetmyname) [inline python/sglang/srt/hardware_backend/npu/quantization/fused_moe_method_npu.py]: add comments

**Review by ping1jing2 — APPROVED** [review]
